target/linux/ipq806x/base-files/etc/board.d/02_network

```diff
@@ -60,6 +60,9 @@ qcom,ipq8064-db149)
 	ucidef_add_switch "switch0" \
 		"1:lan" "2:lan" "3:lan" "4:lan" "6u@eth1" "5:wan" "0u@eth0"
 	;;
+ubnt,unifi-ap-hd)
+	ucidef_set_interface_lan "eth1 eth0"
+	;;
```

Comment:
this is a picky detail, but the interfaces might as well be listed in order `"eth0 eth1"`

Reply:
I have to re-check that this does not result in the wrong MAC address for the LAN bridge. The ETH port labeled "primary" and has the label's MAC address is detected as eth1.

Reply:
the MAC for `br-lan` is insignificant, as long as all ethX have a vendor MAC and `label-mac-device` is correct

however, you might find a way to swap eth0 and eth1 in DTS somehow...

btw what is `qca,pcs-chanid`

Reply:
No idea, it's a leftover from copying the nodes over from vendor DTS. Removed the property and swapped the interface names.